### DISCUSSION TITLE
Kamp enabling

### DISCUSSION THREAD

**Discussion by dasb00ter** [bassamanator/Sovol-SV06-firmware#161] [Q&A] [answered]
@bassamanator didnt think this was worth a pull request b/c maybe I just dont have a basic understanding.
I noticed in the directions it mentions enabling KAMP, however, it seems to be a list of warnings and enabling some options but enabling KAMP itself seems to be missing?I would expect  I would be uncommenting something?

**Comment by bassamanator** [comment, marked as answer]
Good point. I need to adjust the wording, but you'll find settings related info [What are some settings that I can change?](https://github.com/bassamanator/Sovol-SV06-firmware?tab=readme-ov-file#what-are-some-settings-that-i-can-change).

`How do I enable KAMP (Klipper Adaptive Meshing & Purging)?` needs to be changed to `What do I need to know about KAMP`, or something along those lines.
